package.json

```diff
@@ -96,6 +96,7 @@
     "isomorphic-fetch": "2.2.1",
     "luxon": "1.2.0",
     "markdown-it": "^8.4.1",
+    "mockdate": "^2.0.2",
```

Comment:
=> devDependencies

Reply:
I found it weird as well, but it gave me error saying I should add it to `dependencies` instead of `devDependencies` when I tried import in test file.

Now it doesn't give error anymore.